extra_docs/dataset/README.md

```diff
@@ -0,0 +1,269 @@
+# Datasets Module
+A Dataset is used to manipulate records.
+
+### [Initialize](#init) 
+To use dataset features, just add `dataOperations` module to the client initialization. Notice, that for Node.js
+you have to import `dataOperations` from `jexia-sdk-js/node`, but if you are running your application in a browser, you should import it from `jexia-sdk-js/browser`. That is because file format is different under these two platforms.
```

Comment:
`That is because file format is different under these two platforms` - copied from the filesets? ;-)